packages/bloodhound-python/PKGBUILD

```diff
@@ -0,0 +1,38 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+_pkgname='BloodHound.py'
+pkgname='bloodhound-python'
+pkgver=v1.0.1.r8.gefc8b57
+pkgrel=1
+pkgdesc='Bloodhound python data collector'
+arch=('any')
+groups=('blackarch' 'blackarch-recon' 'blackarch-windows')
+url='https://github.com/fox-it/BloodHound.py'
+license=('MIT')
+depends=('python' 'python-dnspython' 'impacket' 'python-ldap3' 'python-pyasn1'
+         'python-future' 'ldapdomaindump')
+makedepends=('python-setuptools' 'git')
+source=("git+https://github.com/fox-it/$_pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $_pkgname
+
+  git describe --long | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
+}
+
+build() {
+  cd "$_pkgname"
+
+  python setup.py build
+}
+
+package() {
+  cd "$_pkgname"
+
+  python setup.py install --root="$pkgdir" --prefix=/usr -O1 --skip-build
+
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" README.md
+  install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+}
```

Comment:
Please add final newline.

Reply:
This silly editor that remove it :/

Reply:
Yes, you need to add two newlines really. Thanks for the fix.